packages/xstate-inspect/package.json

```diff
@@ -38,23 +38,28 @@
     "url": "https://github.com/statelyai/xstate/issues"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.0.0",
+    "@rollup/plugin-node-resolve": "^11.0.1",
     "@types/ws": "^8.2.2",
+    "fast-safe-stringify": "^2.0.7",
     "rollup": "^2.69.0",
+    "rollup-plugin-replace": "^2.2.0",
+    "rollup-plugin-terser": "^5.1.2",
     "rollup-plugin-typescript2": "^0.30.0",
     "typescript": "^4.5.2",
     "ws": "^8.4.0",
     "xstate": "*"
   },
   "peerDependencies": {
     "@types/ws": "^8.0.0",
-    "ws": "^8.0.0"
+    "fast-safe-stringify": "^2.0.7",
```

Comment:
This should be moved back to dependencies as the `createUmdConfig` should "bundle" this in the final file.